webapp/graphite/render/glyph.py

```diff
@@ -226,19 +226,21 @@ def applySettings(self, axisMin=None, axisMax=None, axisLimit=None):
   def makeLabel(self, value):
     value, prefix = format_units(value, self.step, system=self.unitSystem)
     span, spanPrefix = format_units(self.span, self.step, system=self.unitSystem)
+    if prefix:
+      prefix += " "
     if value < 0.1:
       return "%g %s" % (float(value), prefix)
     elif value < 1.0:
       return "%.2f %s" % (float(value), prefix)
     if span > 10 or spanPrefix != prefix:
       if type(value) is float:
-        return "%.1f %s " % (value, prefix)
+        return "%.1f %s" % (value, prefix)
       else:
-        return "%d %s " % (int(value), prefix)
+        return "%d %s" % (int(value), prefix)
     elif span > 3:
-      return "%.1f %s " % (float(value), prefix)
+      return "%.1f %s" % (float(value), prefix)
     elif span > 0.1:
-      return "%.2f %s " % (float(value), prefix)
+      return "%.2f %s" % (float(value), prefix)
```

Comment:
@cbowman0: your suggested patch didn't include this last change. I think this was an oversight, so I removed the space here, too.